applications/GeoMechanicsApplication/tests/cpp_tests/custom_utilities/test_process_utilities.cpp

```diff
@@ -141,4 +150,95 @@ static const std::vector<NamedFactory> ProcessesAndOperations = {
 
 INSTANTIATE_TEST_SUITE_P(ProcessAndOperationUtilitiesTests, ModelPartsTest, ::testing::ValuesIn(ProcessesAndOperations));
 
+KRATOS_TEST_CASE_IN_SUITE(AddProcessesSubModelPartList_AddsEmptyList, KratosGeoMechanicsFastSuiteWithoutKernel)
```

Comment:
Perhaps we can make the test case name slightly more accurate, e.g.:
```suggestion
KRATOS_TEST_CASE_IN_SUITE(AddProcessesSubModelPartList_AddsEmptyListWhenThereAreNoProcesses, KratosGeoMechanicsFastSuiteWithoutKernel)
```

Reply:
changed